src/applications/ask-va/containers/ResponseInboxPage.jsx

```diff
@@ -129,64 +110,105 @@ const ResponseInboxPage = ({ loggedIn }) => {
-        </p>
-        <p>
-          <strong>{RESPONSE_PAGE.STATUS}</strong>{' '}
-          {inboxMessage.attributes.processingStatus}
-        </p>
-        <h2 className="vad-u-margin-top--0">{RESPONSE_PAGE.YOUR_QUESTION}</h2>
+      <BreadCrumbs currentLocation={location.pathname} />
+      <div className="usa-width-two-thirds medium-8 columns vads-u-padding--0">
+        <h1 className="">{RESPONSE_PAGE.QUESTION_DETAILS}</h1>
 
-        <p className="vads-u-margin-top--2p5 vads-u-margin-bottom--5">
-          {inboxMessage.attributes.question}
-        </p>
+        <div className="vads-u-margin-bottom--3">
+          <div>
+            <span className="usa-label">{inquiryData.attributes.status}</span>
+          </div>
+
+          <div className="vads-u-margin-top--2">
+            <p className="vads-u-margin--0">
+              <span className="vads-u-font-weight--bold">Submitted:</span>{' '}
+              {formatDate(inquiryData.attributes.createdOn)}
+            </p>
+            <p className="vads-u-margin--0">
+              <span className="vads-u-font-weight--bold">Last updated:</span>{' '}
+              {formatDate(inquiryData.attributes.lastUpdate)}
+            </p>
+            <p className="vads-u-margin--0">
+              <span className="vads-u-font-weight--bold">Category:</span>{' '}
+              {inquiryData.attributes.category}
+            </p>
+            <p className="vads-u-margin--0">
+              <span className="vads-u-font-weight--bold">
+                Reference number:
+              </span>{' '}
+              {inquiryData.attributes.inquiryNumber}
+            </p>
+          </div>
+        </div>
+
+        {/* Temporarily hidden for research study
         <h2 className="vad-u-margin-top--0">{RESPONSE_PAGE.ATTACHMENTS}</h2>
         {inboxMessage.attributes.attachments.length === 0
           ? emptyMessage(RESPONSE_PAGE.NO_ATTACHMENTS)
           : inboxMessage.attributes.attachments.map(attachment => (
               <div key={attachment.id}>{attachmentBox(attachment.name)}</div>
             ))}
-        <hr />
-        <h2 className="vads-u-margin-y--1p5">{RESPONSE_PAGE.INBOX}</h2>
-        {inboxMessage.attributes.reply.data.length === 0 ? (
+        <hr /> */}
+        <div className="vads-l-row vads-u-justify-content--space-between vads-u-align-items--baseline">
+          <h2 className="vads-u-margin-y--2">
+            {RESPONSE_PAGE.YOUR_CONVERSATION}
+          </h2>
+          <button
+            className="vads-u-display--flex vads-u-flex-direction--row vads-u-align-items--center va-button-link vads-u-text-decoration--none"
+            type="button"
+            onClick={() => window.print()}
+          >
+            <span className="vads-u-color--primary">
+              <VaIcon
+                size={3}
+                icon="print"
+                className="vads-u-margin-right--1"
+                srtext="icon representing print"
+                aria-hidden="true"
```

Comment:
If this icon is purely decorative, then you should remove the `srtext` prop. The web component will then automatically add `aria-hidden` internally so you can remove that here too.

https://design.va.gov/storybook/?path=/docs/uswds-va-icon--docs

Reply:
Noted for the follow up ticket for Dashboard